templates/legal/companies.html

```diff
@@ -99,9 +97,9 @@ <h1>Canonical companies</h1>
             <td>Canonical Japan KK</td>
             <td>
               <ul class="p-list">
-                <li class="p-list__item">c/o 5th Floor, Blue Fin Building</li>
-                <li class="p-list__item">110 Southwark Street</li>
-                <li class="p-list__item">London SE1 0SU</li>
+                <li class="p-list__item">c/o Canonical</li>
+                <li class="p-list__item">The Office Group, St Dunstans House,</li>
+                <li class="p-list__item">London EC4A 3TW,</li>
```

Comment:
the address is for the contactable address but the post code is for the registered address